.github/workflows/kover.yml

```
@@ -0,0 +1,151 @@
+# .github/workflows/diff-coverage.yml
+name: Diff Coverage (Kover + diff-cover)
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)